pages/opam-using.html

```diff
@@ -70,4 +87,64 @@ <h2>Using OPAM to install Coq packages</h2>
 
 <pre><code>opam install coq-sudoku
 </code></pre>
+
+<h2>Advanced usage</h2>
+
+<h3>Managing different versions of OCaml and Coq</h3>
+
+<p>By default, opam will use the global OCaml installation. Opam can handle
+different versions of OCaml and other packages (including Coq) via
+<em>switches</em> or <em>roots</em>.</p>
+
+<h4>Switches</h4>
+
+<p>Switches provide separate environments, with their own versions of OCaml and
+installed packages.</p>
+
+<p>The following command creates a switch named <code>with-coq</code> with OCaml
+<#OCAMLV>:</p>
+
+<pre><code># Run one of the following depending on your version of opam
+opam switch create with-coq <#OCAMLV>
+</code></pre>
+
+<p>Change to an existing switch named <code>other-switch</code> with this command:</p>
+
+<pre><code>opam switch other-switch
+eval $(opam env)
```

Comment:
Is it actually needed to repeat this line in the case of switches?

Reply:
yes